web_accounts_list.json

```diff
@@ -4591,6 +4591,61 @@
            "known_accounts" : ["john", "junech"],
            "category" : "dating",
            "valid" : true
+          },
+          {
+           "name" : "Tunisie-Foot",
+           "check_uri" : "https://www.tunisie-foot.com/forum/wcf/user/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "profile",
+           "account_missing_string" : "Page Non Trouvée - Tunisie-Foot",
+           "account_missing_code" : "404",
+           "known_accounts" : ["7701-manager", "28487-tunisia-gang-city"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "ICQ Chat",
+           "check_uri" : "https://icq.icqchat.co/members/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "person",
+           "account_missing_string" : "Oops! We ran into some problems",
+           "account_missing_code" : "404",
+           "known_accounts" : ["brookenora.54", "bigdaddy.77"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "HeyLink.me",
+           "check_uri" : "https://heylink.me/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "username",
+           "account_missing_string" : "We can't find the page that you're looking for :(",
+           "account_missing_code" : "404",
+           "known_accounts" : ["mohammed13", "johnny"],
+           "category" : "misc",
+           "valid" : true
+          },
+          {
+           "name" : "Yazawaj",
+           "check_uri" : "https://www.yazawaj.com/profile/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "dating profile",
+           "account_missing_string" : "الصفحة المطلوبة غير موجودة",
+           "account_missing_code" : "404",
+           "known_accounts" : ["rachid007", "LordMohy"],
+           "category" : "dating",
+           "valid" : true
+          },
+          {
+           "name" : "Alloannonces",

```

Comment:
This one appears to redirect the browser (HTTP 301 response) so please check the URI used. Also the detection string might be off.